### PR TITLE
Release 346 remove locking

### PR DESCRIPTION
**Pull request by ghost**
@drozdo @macbre

### CODE REVIEW

File: includes/Revision.php [hunk 1]
```diff
@@ -328,7 +328,7 @@ private static function fetchFromConds( $db, $conditions, $flags = 0 ) {
 			self::selectUserFields()
 		);
 		$options = array( 'LIMIT' => 1 );
-		if ( $flags & self::READ_LOCKING ) {
+		if ( ( $flags & self::READ_LOCKING ) == self::READ_LOCKING ) {
 			$options[] = 'FOR UPDATE';
```
Comment (drozdo) [inline includes/Revision.php]: LOCK IN SHARE MODE

**Comment by drozdo** [comment]
(y)

**Comment by drozdo** [comment]
LGTM

**Comment by ghost** [comment]
:8ball:

**Pull request merged**